pkgs/applications/networking/browsers/lagrange/default.nix

```diff
@@ -13,37 +13,36 @@
 , pcre
 , SDL2
 , AppKit
+, zip
 , zlib
 }:
 
 stdenv.mkDerivation rec {
   pname = "lagrange";
-  version = "1.7.3";
+  version = "1.8.0";
 
   src = fetchFromGitHub {
     owner = "skyjake";
     repo = "lagrange";
     rev = "v${version}";
-    sha256 = "sha256-peBdmz/aucrKO5Vsj8WkHkpGpLm4inQHee133Zph3MM=";
+    sha256 = "sha256-wZ0DyQq54CVClagx72rsobo5x/gNrB1lDXoXEsV63Xg=";
     fetchSubmodules = true;
   };
 
   postPatch = ''
     rm -r lib/fribidi lib/harfbuzz
+
+    sed -i 's|AND EXISTS.*/lib/fribidi||' Depends.cmake
+    sed -i 's|AND EXISTS.*/lib/harfbuzz/CMakeLists.txt||' Depends.cmake
```

Comment:
Can we use substituteInPlace?

Reply:
I've tried `substituteInPlace`, but I don't know how propery escape `${CMAKE_...}`. Neither `\${CMAKE_...}`, nor `\$\{CMAKE_...\}` works for me.

Reply:
I hope `Depends.cmake` will be fixed in upstream anyway.

Reply:
To escape it for nix you would need ``''${CMAKE``. Maybe the bash part needs more,

Reply:
`''$` doesn't work for me in this case...

Reply:
Reported upstream: https://github.com/skyjake/lagrange/issues/390

Reply:
Fixed in 1.8.3, we don't need this patch anymore.